client/js/components/map/admin/LayerSettings.vue

```diff
@@ -413,9 +413,12 @@ export default {
 
       externalApi(url)
         .then(response => {
+          return response.text()
+        })
+        .then(text => {
```

Comment:
```suggestion
        .then(capabilities => {
```
Please use a more descriptive variable name. I think text is too generic and makes it harder to understand.